packages/web/src/components/now-playing/NowPlaying.tsx

```diff
@@ -206,7 +203,7 @@ const NowPlaying = g(
     } else {
       displayInfo = {
         title: collectible?.name as string,
-        track_id: collectible?.id as string,
+        track_id: Number(collectible?.id),
```

Comment:
This seems a little sus. Are these really stringified integers or do they need to be `HashId.parse()`d?